lntest/itest/lnd_test.go

```diff
@@ -1566,14 +1566,11 @@ func testPaymentFollowingChannelOpen(net *lntest.NetworkHarness, t *harnessTest)
 
 	// Send payment to Bob so that a channel update to disk will be
 	// executed.
-	sendAndAssertSuccess(
-		t, net.Alice,
-		&routerrpc.SendPaymentRequest{
-			PaymentRequest: bobPayReqs[0],
-			TimeoutSeconds: 60,
-			FeeLimitSat:    1000000,
-		},
-	)
+	sendAndAssertSuccess(t, net.Alice, &routerrpc.SendPaymentRequest{
```

Comment:
I've seen mixed styles of,
```go
foo(a, b
   c, d)
```
and
```go
foo(
   a,
   b,
   c,
)
```

According to the [guideline](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#48-additional-style-constraints-on-top-of-gofmt), we should stick to the latter one?

Reply:
This is a special case for struct initializations in function calls.
If it was just `sendAndAssertSuccess(t, net.Alice, &routerrpc.SendPaymentRequest{})` it would fit on a single line.
The curly braces of the struct gives it a new indentation context.
Looking through the golang codebase, it seems this is used all over, for example: https://github.com/golang/go/blob/master/src/image/gif/writer.go#L468

Reply:
Cool. I actually like your approach, just want to make sure. Maybe the guideline could be more specific.

Reply:
Yes, this could be mentioned more explicitly in the guidelines. Though we also just try to stick to common golang conventions. Maybe there is a good document out there that we could link to?